### PR TITLE
CYBER-1458 Add automated certificate rotation policy

### PR DESCRIPTION
**Pull request by timja**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CYBER-1458

### Change description ###

Sourced from: https://ministryofjustice.github.io/security-guidance/automated-certificate-renewal/#automated-certificate-renewal

Document proposed policy for automated certification rotation

image

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by cakeben — APPROVED** [review]

File: source/information-security/certificate-automation.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Where technically suitable, all new HMCTS domains must use automated certificate techniques and services, 
+such as [Microsoft managed certificates](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-custom-domain-https#option-1-default-use-a-certificate-managed-by-front-door)
+(most preferred) or LetsEncrypt (uses ACME)
+
+Over time, existing HMCTS domains must also be considered for migration to automated certificate provisioning and 
+management techniques (preferably on their next certificate renewal cycle in advance of expiry) in order to reduce 
+the consequences and management overheads of manual certificate renewal.
+
+HMCTS acknowledges that not all systems support automated certificate management.
+but leveraging such technology where possible reduces management overheads, the costs of such overheads and the consequences of
```
Comment (njrich28) [inline source/information-security/certificate-automation.html.md.erb:21]:
```suggestion
HMCTS acknowledges that not all systems support automated certificate management,
but leveraging such technology where possible reduces management overheads, the costs of such overheads and the consequences of 
```

File: source/information-security/certificate-automation.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+HMCTS acknowledges that not all systems support automated certificate management.
+but leveraging such technology where possible reduces management overheads, the costs of such overheads and the consequences of 
+unexpected certificate expiry.
+
+## Manual certificate requests
+
+Where automated certificate renewal is not possible, new certificates must be acquired through the DTS Operations Management team.
+
+To request a manually issued certificate, complete the [certificate request form](https://tools.hmcts.net/confluence/display/RD/Requesting+SSL+Certificates) and send it,
```
Comment (njrich28) [inline source/information-security/certificate-automation.html.md.erb:28]: Is it normal to link to closed documentation? I'm presuming we've done it before but feels dirty

**Review by njrich28 — APPROVED** [review]
Couple of minor things - happy to approve content. We prob need to get this approved by someone from IA/SecOps before publishing.

File: source/information-security/certificate-automation.html.md.erb (hunk 2 above)
Comment (timja) [reply source/information-security/certificate-automation.html.md.erb:28]: the MoJ docs links off to it in some places, even in the docs I'm copying from the SSL form is not open.

They have a disclaimer on their main site that says they are in the process of moving from their intranet to their github site but that in the meantime some docs will be linked off to

**Review by adusumillipraveen — APPROVED** [review]

**Review by subseven-oax — APPROVED** [review]
Security team has reviewed this and it has been approved.

**Pull request merged**